package.json

```diff
@@ -24,7 +24,9 @@
     "@ionic-native/core": "5.0.0-beta.21",
     "@ionic-native/splash-screen": "5.0.0-beta.21",
     "@ionic-native/status-bar": "5.0.0-beta.21",
-    "@ionic/angular": "4.0.0-beta.15",
+    "@ionic/angular": "4.0.0-beta.13",
+    "@ionic/storage": "^2.2.0",
```

Comment:
Don't need to use storage to save data to local storage now. Chaw is working on the storage service, we should implement the storage logic after chaw finished that service. That means we will use a shared service to save data into local storage, instead of importing "Storage" in every component